src/scss/utilities/index.scss

```diff
@@ -1,9 +1,9 @@
 @charset 'UTF-8';
 
 // First import decanter as no markup.
-//@import 'decanter/core/src/scss/decanter-no-markup';
+@import 'decanter/core/src/scss/decanter-no-markup';
 // Import full scss version of decanter to use variables etc
-@import 'decanter/core/src/scss/decanter';
+//@import 'decanter/core/src/scss/decanter';
```

Comment:
We try to avoid importing the full version of Decanter in the utility section as that is supposed to be able to be included without outputting anything by default.